### PR TITLE
Make MUV notebook run to completion

### PR DESCRIPTION
**Pull request by rbharath**
Debugs the MUV example notebook and gets it to run to completion. Will add more details of changes once ready to merge.

### CODE REVIEW

**Comment by rbharath** [comment]
Changes in this PR:

1. Removes some extraneous print statements from `featurizers/featurize.py`.
2. Adds `verbosity` to `TensorflowGraph`. Fixes `TensorflowGraph.fit()` to respect `nb_epoch`.
3. Adds new test set `hyperparameters/tests/test_tf_hyperparam_opt.py` to test use of `HyperparamOpt` with `TensorflowModel`s.
